backend/tests/test_all.py

```
@@ -0,0 +1,228 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)